src/browser/documentation/help/keys.tsx

```diff
@@ -31,80 +31,201 @@ const category = 'browserUiCommands'
 const content = (
   <>
     <table className="table-condensed table-help table-help--keys">
-      <thead>
+      <tbody>
         <tr>
-          <th>Editor action</th>
-          <th>Any mode</th>
-          <th>Single-line mode</th>
+          <th>Global actions</th>
+          <th />
+          <th />
         </tr>
-      </thead>
-      <tbody>
         <tr>
-          <td>Execute current command</td>
+          <td>Change focus to editor</td>
           <td>
-            <div className="key code">
-              {isMac ? '<Cmd-Return>' : '<Ctrl-Return>'}
-            </div>
+            <div className="key code">{printShortcut(FOCUS_SHORTCUT)}</div>
           </td>
+        </tr>
+        <tr>
+          <td>Toggle fullscreen editor</td>
           <td>
-            <div className="key code">{'<Return>'}</div>
+            <div className="key code">{printShortcut(FULLSCREEN_SHORTCUT)}</div>
           </td>
         </tr>
+        <tr>
+          <td />
+        </tr>
+      </tbody>
+    </table>
+
+    <table className="table-condensed table-help table-help--keys">
+      <thead>
+        <tr>
+          <th title="Field #1" className="padding30">
+            Editor action
+          </th>
+          <th title="Field #2" className="padding30">
+            Windows / Linux
+          </th>
+          <th title="Field #3" className="padding30">
+            Mac
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Execute current command</td>
+          <td>Ctrl + Enter</td>
```

Comment:
You can use the isMac function to only display the relevant keys here

Reply:
I'll keep it as it is for now, lots of manual work. But yes it should be fixed at some point